bin/functional-tests/test_config.py

```diff
@@ -52,13 +52,13 @@ def test_houston_config(houston_api):
 
 
 def test_houston_can_reach_prometheus(houston_api):
```

Comment:
A PyTest feature that we may want to benefit from in the integration tests are the custom markers:
https://docs.pytest.org/en/6.2.x/example/markers.html

This would allow developers to run / not run specific tests (eg. Houston API or Nginx)

Reply:
i think it's better to leave them as separate test cases. they call different helpers and use different args. we can already select for test specific test cases to run with `-k` notation.

Reply:
we were starting from very low confidence when we started with pytest chart testing, so running all versions made sense. it might make sense to default to not testing all versions of k8s, except in specific circumstances. i'll have to think more about that as these tests evolve. thanks for the feedback on that!

also i agree that having `-k` has served us pretty well. eg: `-k '1.21 and configmap_with_addition_alerts'` would only run one test even with the parametrized versions. i'm trying to think of a use case where having additional markers would help...